vector/src/main/java/im/vector/app/features/notifications/PushRuleTriggerListener.kt

```diff
@@ -32,44 +32,32 @@ class PushRuleTriggerListener @Inject constructor(
 
     private var session: Session? = null
 
-    override fun onMatchRule(event: Event, actions: List<Action>) {
-        Timber.v("Push rule match for event ${event.eventId}")
-        val safeSession = session ?: return Unit.also {
-            Timber.e("Called without active session")
-        }
+    override fun onEvents(pushEvents: PushEvents) {
+        session?.let { session ->
+            val notifiableEvents = createNotifiableEvents(pushEvents, session)
+            notificationDrawerManager.updateEvents { queuedEvents ->
```

Comment:
all notification updates from the sync response are now batched together into a single update